metecho/api/models.py

```diff
@@ -124,7 +124,9 @@ class User(PushMixin, HashIdMixin, AbstractUser):
     currently_fetching_repos = models.BooleanField(default=False)
     currently_fetching_orgs = models.BooleanField(default=False)
     organizations = models.ManyToManyField(
-        "api.GitHubOrganization", related_name="users"
+        "api.GitHubOrganization",
+        related_name="users",
+        blank=True,
```

Comment:
@jerivas I think this was maybe overlooked initially? I wasn't able to save a User in the admin unless they belonged to an Org.

Reply:
Ah yes, definitely an oversight